latextools_utils/cache.py

```diff
@@ -0,0 +1,218 @@
+import os
+import pickle
```

Comment:
It may be better to use cPickle here for those on ST2. It's substantially faster. Just need to do something like:

```python
try:
    import cPickle as pickle
except ImportError:
    import pickle
```